ajax/libs/angular-parse/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "angular-parse",
+  "version": "0.2.3",
+  "description": "Angular wrapper for Parse.com JavaScript SDK.",
+  "license": "MIT",
+  "author": "<ivnivnch@gmail.com>",
+  "filename": "angular-parse.js",
+  "keywords": [
+    "angular",
+    "angularjs",
+    "parse",
+    "parsesdk"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ivnivnch/angular-parse.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ivnivnch/angular-parse.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "!(*.iml)",
+          "angular-parse*"
+        ]
```

Comment:
Let's use 
```js
        "files": [
          "angular-parse*.+(js|css|map)"
        ]
```
Because we don't excluded pattern in root path.
As usual, we use excluded pattern in `dist`, `build`...folder, and use included pattern in `""`, `src`...etc.
Thanks.